apb.yml

```diff
@@ -12,10 +12,48 @@ metadata:
      Import Virtual Machine from an existing image or a VMware environment.
 plans:
   - name: vmware
-    description: Import a virtual machine from VMware vCenter
+    description: Import a virtual machine from VMware vCenter.
     free: True
     metadata:
-      displayName: Import from VMware
+      displayName: b. Import from VMware.
```

Comment:
Let's keep the URL ones together as A and B

Reply:
Done